src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.c

```diff
@@ -304,7 +304,7 @@ void EnBji01_DialogueHandler(EnBji01* this, GlobalContext* globalCtx) {
 
 void func_809CD634(EnBji01* this, GlobalContext* globalCtx) {
     func_801A5BD0(0x6F);
-    func_801A89A8(0xE0000101);
+    Audio_QueueSeqCmd(0xE0000101);
```

Comment:
@rylieb will be interested to see this function name, I think. I remember us discussing these magic numbers before.

Reply:
How these are handled is an audio WIP, but to give an idea of what the general approach will look like, I'm thinking of following this approach with how it's done in OoT atm:
https://github.com/zeldaret/oot/blob/b9ad6aac524242a633927f8e0bfc426c6d4f87cb/src/code/code_800F9280.c#L16-L25